api/swagger-spec/v1.json

```diff
@@ -19399,6 +19399,10 @@
      "data": {
       "type": "object",
       "description": "Data contains the configuration data. Each key must consist of alphanumeric characters, '-', '_' or '.'."
+     },
+     "binaryData": {
+      "type": "object",
+      "description": "Data contains the binary data. Each key must consist of alphanumeric characters, '-', '_' or '.'."
```

Comment:
Better description here, and improve the existing `data` description.  `binaryData contains configuration data that is not required to be UTF-8 and ...`.  Describe how the two fields validate together.  Describe why I would use one or the other.

Reply:
Done